aio_pika/channel.py

```diff
@@ -79,35 +79,51 @@ def return_callbacks(self) -> CallbackCollection:
 
     @property
     def is_closed(self):
-        if not self._channel:
+        if not self._channel or self._is_closed_by_user:
             return True
-        return self.channel.is_closed
+        return self._channel.is_closed
+
+    @property
+    def _is_closed_by_user(self):
```

Comment:
Maybe it just worth to add `_is_closed_by_user` attribute for the `Channel` class? I mean changing it manually without all this magic with empty tuple..

Reply:
Scheduled on 7.0 in #436